livestyled/client.py

```diff
@@ -418,13 +379,64 @@ def get_news(
             params=filter_params
         )
 
-    def create_news(
-            self,
-            attributes: Dict,
-    ) -> Dict:
+    def create_news(self, attributes: Dict) -> Dict:
         payload = NewsSchema().dump(attributes)
         new_team = self._api_post(
             'v4/{}'.format(NewsSchema.Meta.url),
             payload
         )
         return NewsSchema().load(new_team)
+
+    def get_merchant_account(self, id: int or str) -> Generator[Dict, None, None]:
+        return self._get_resource(
+            id,
+            MerchantAccountSchema,
+        )
+
+    def get_merchant_accounts(self, params: dict or None) -> Generator[Dict, None, None]:
+        return self._get_resources(
+            MerchantAccountSchema,
+            params=params
+        )
+
+    def create_merchant_account(self, attributes: Dict) -> Dict:
+        payload = MerchantAccountSchema().dump(attributes)
+        merchant_account = self._api_post(
+            'v4/{}'.format(MerchantAccountSchema.Meta.url),
+            payload
+        )
+        return MerchantAccountSchema().load(merchant_account)
+
+    def get_payment_gateway(self, id: int or str) -> Generator[Dict, None, None]:
+        return self._get_resource(
+            id,
+            PaymentGatewaySchema,
+        )
+
+    def get_payment_gateways(self, params: dict or None) -> Generator[Dict, None, None]:
```

Comment:
should we call `params` `filters` like other method. It makes it easy to know it is API V4 filters. It just reads well. What do you think?